File: 4 - listas/lista.py
import random

def exibir(lista):
    for item in lista:
        print(item)

lista = []
for i in range(100):
    #valor = int(input('Digite um numero inteiro: '))
    valor = random.randint(0,400)
    if (valor not in lista):
        lista.append(valor)
        lista.sort()

print("Tamanho da lista ", len(lista))
exibir(lista)

File: 4 - listas/Lista.java
import java.util.List;
import java.util.Scanner;
import java.util.ArrayList;

public class Lista {
    public static void main(String a[]) {
        List<Integer> lista = new ArrayList<Integer>();
        Scanner teclado = new Scanner(System.in);
        int valor;
        for (int i = 0; i < 4; i++) {
            System.out.print("Digite um numero inteiro: ");
            valor = teclado.nextInt();
            if (!lista.contains(valor)) {
                lista.add(valor);
                lista.sort(null);
            }
        }
        System.out.println(lista);
    }
}